packages/lesswrong/components/votes/lwReactions/NamesAttachedReactionsVoteOnComment.tsx

```diff
@@ -168,48 +177,44 @@ export const useNamesAttachedReactionsVoting = (voteProps: VotingProps<VoteableT
     })
   }
 
-  function toggleReaction(name: string, quote?: string) {
+  function toggleReaction(name: string, quote: QuoteLocator|null) {
     if (!currentUser) {
       openLoginDialog();
       return;
     }
-    const currentUserReaction = getCurrentUserReaction(name);
-    const shouldClearUserReaction = getCurrentUserReactionVote(name) && (!quote || currentUserReaction?.quotes?.includes(quote))
+    const shouldClearUserReaction = !!getCurrentUserReactionVote(name, quote);
 
     if (shouldClearUserReaction) {
-      if (quote && currentUserReaction) {
-        clearQuoteFromCurrentUserReaction(currentUserReaction, quote);
-      } else {
-        clearCurrentUserReaction(name);
-      }
+      clearCurrentUserReaction(name, quote);
     } else {
       const initialVote = "created"; //TODO: "created" vs "seconded"
       addCurrentUserReaction(name, initialVote, quote);
     }
   }
   
-  function addCurrentUserReaction(reactionName: string, vote: VoteOnReactionType, quote?: string) {
+  function addCurrentUserReaction(reactionName: EmojiReactName, vote: VoteOnReactionType, quote: QuoteLocator|null) {
     if (!currentUser) {
       openLoginDialog();
       return;
     }
+    
+    // TODO
```

Comment:
Does this need any follow-up?

Reply:
Looks like a remnant and the thing it was saying needed doing is already done. Will remove.